yarn.lock

```diff
@@ -3524,7 +3524,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"axios@npm:^1.6.0, axios@npm:^1.7.2, axios@npm:^1.7.3":
+"axios@npm:^1.6.0, axios@npm:^1.7.2":
```

Comment:
Theres are some dependencies missing.

Reply:
I added tempwise a resolution for axios because we are currently stuck on an old nx version.